test/ngRoute/routeSpec.js

```diff
@@ -65,8 +65,8 @@ describe('$route', function() {
       $httpBackend.when('GET', 'Chapter.html').respond('chapter');
       $httpBackend.when('GET', 'test.html').respond('test');
       $httpBackend.when('GET', 'foo.html').respond('foo');
-      $httpBackend.when('GET', 'baz.html').respond('baz');
       $httpBackend.when('GET', 'bar.html').respond('bar');
+      $httpBackend.when('GET', 'baz.html').respond('baz');
```

Comment:
Does this change have any effect?

Reply:
No afaict :grin:
<sub>(Except for ordering definitions in proper foo-bar-baz order, which is as great an effect as a change can have 😇)</sub>